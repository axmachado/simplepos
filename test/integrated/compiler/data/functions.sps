module mod1;

int x, y;

x = 1;

y = f1(x);

int f1 (int a) {
    return a+1;
}
