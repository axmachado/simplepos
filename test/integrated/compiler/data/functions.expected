mod1 ()
{
    x = 1
    y = f1 (x)
}

int f1 (int a)
{
    return (a + 1)
}
